packages/storybook/stories/components/quantity/migration.from.17.x.mdx

```diff
@@ -17,95 +17,95 @@ You can use the new `isDisabled` attribute to obtain the same behavior.
 
```

Comment:
would be nice to add a `## Usage changes` section first that explain how the component did move from a full-slot solution to a all-in-one component (see button or tooltip for example)